tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -425,7 +433,27 @@ StatusOr<ExecutionOutput> GpuExecutable::ExecuteAsyncOnStream(
     const ServiceExecutableRunOptions* run_options,
     std::vector<ExecutionInput> arguments,
     HloExecutionProfile* hlo_execution_profile) {
-  XLA_SCOPED_LOGGING_TIMER(absl::StrCat("GpuExecutable::ExecuteAsyncOnStream(",
+  return ExecuteAsyncOnStreamImpl(run_options, absl::MakeSpan(arguments),
```

Comment:
This depends on whether you need the argument to be non-const. If you can make it `absl::Span<ExecutionInput const>`, you won't need `absl::MakeSpan`

Reply:
I already tried. But we can't make it const. MarkToBeReleasedArguments change the values.